src/serializer.js

```diff
@@ -14,12 +14,23 @@ function createSerializer(styleSheet) {
     val.withStyles = true
     const printedVal = printer(val)
     if (styles) {
-      return `${styles}\n\n${printedVal}`
+      return replaceClassNames(selectors, styles, printedVal)
     } else {
       return printedVal
     }
   }
 
+  function replaceClassNames(selectors, styles, code) {
+    let index = 0
+    return selectors.reduce((acc, className) => {
+      const escapedRegex = new RegExp(
```

Comment:
Would this be best within a `replaceClassNames.js` or as part of a `utils.js` (or other)?